apps/fluent-tester/src/FluentTester/TestComponents/Avatar/BasicAvatar.tsx

```diff
@@ -100,12 +100,12 @@ export const StandardUsage: FunctionComponent = () => {
         <StyledPicker prompt="Avatar Color" selected={avatarColor} onChange={onAvatarColorChange} collection={avatarColors} />
         <StyledPicker prompt="Presence status" selected={presence} onChange={onPresenceChange} collection={allPresences} />
```

Comment:
Possibly out of scope for this PR, but could we _not_ use Picker? That's going to stop working very soon.

Reply:
I've been thinking about it. Because I'm looking forward to RN update. I'll try to fix it in scope of this PR